plugins/node/opentelemetry-instrumentation-cassandra/test/cassandra-driver.test.ts

```diff
@@ -228,6 +253,38 @@ describe('CassandraDriverInstrumentation', () => {
         assertSingleSpan('cassandra-driver.execute', query.substr(0, 25));
       });
     });
+
+    describe('responseHook', () => {
```

Comment:
Can you please also add a test where the hook throws, to assert the exception is handled and not affecting the user?
You can see many examples in other instrumentation tests

Reply:
Of course! I added a test to make sure hook exceptions do not affect span creation or user flow